next/utils/getBranchInfo.ts

```diff
@@ -0,0 +1,15 @@
+import { BranchPlaceNestedEntityFragment } from '@bratislava/strapi-sdk-city-library'
+
+export const getBranchInfo = (branch: BranchPlaceNestedEntityFragment | null | undefined) => {
```

Comment:
It's clear what code does, but the "business" reason is not clear.

What is `parentBranch`, why is the address used from there if present? Please add comment.

Reply:
Noted to TODOs